recipes/r-gapminder/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set version = '0.3.0' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-gapminder
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/gapminder_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/gapminder/gapminder_{{ version }}.tar.gz
+  sha256: fb5227cd0c0d3ee3fda70f50c578884e5fc6bfbfc98786bae4cf12c582caec18
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{posix}}zip               # [win]
```

Comment:
I just found a bug, that prevented this line from being remove. Please checkout the latest skeleton helper, sorry.

Reply:
It looks like this line might actually be needed in this case? AppVeyor returns an error 

```
Warning in system(paste(shQuote(ZIP), "-r9Xq", filepath, paste(curPkg, collapse = " "))) :
  '"zip"' not found
running 'zip' failed
```